src/components/Home/Home.jsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import './Home.module.css';
+
+const Home = () => (
+  <main>
+    <h2 style={{
+      padding: '20px 0',
+      lineHeight: '1.9em',
+      color: '#ecc670',
+      textAlign: 'center',
+      fontSize: '2rem',
+    }}
+    >
+      Welcome to our Page!
+    </h2>
+    <p>
+      Discover a world of numbers and inspiration at Math Magicians.
+      Our website offers you two fantastic tools: a feature-rich calculator
+      and a daily dose of humor with our Quote of the Day.
+    </p>
+    <h3>Calculator:</h3>
+    <p>
+      Whether you&apos;re tackling complex math problems or just need a simple calculation,
+      our calculator has you covered. It can handle addition, subtraction,
+      multiplication, division, percentages, and more. Crunch numbers effortlessly
+      with Math Magicians.
+    </p>
+    <h3>Quote of the Day:</h3>
+    <p>
+      Start your day with a smile by checking out our Quote of the Day.
+      We bring you a fresh and funny quote every day to brighten your mood and
+      share with friends. Laughter is the best way to kickstart your day!
+    </p>
+    <p>
+      Explore Math Magicians and make math enjoyable and inspirational.
+      Dive into the world of numbers and humor all in one place. Thank you for
+      choosing Math Magicians for your daily mathematical needs and laughs.
+    </p>
```

Comment:
> - [ ]  Hello @HossainAraf, Please make a small adjustment to the text so that it aligns with the navbar, as shown in the design layout.👍
> 
> ![image](https://github.com/HossainAraf/math-magician/assets/40009354/2f649960-b875-43c6-8183-07d1144963ab)
> 
> ![ox](https://github.com/HossainAraf/math-magician/assets/40009354/96f9c190-8e06-4af4-85f5-7f70e56eb97c)
>